apps/okreads-e2e/src/specs/search-books.spec.ts

```diff
@@ -16,12 +16,17 @@ describe('When: Use the search feature', () => {
     expect(items.length).toBeGreaterThan(1);
   });
 
-  xit('Then: I should see search results as I am typing', async () => {
+  it('Then: I should see search results as I am typing', async () => {
     await browser.get('/');
     await browser.wait(
       ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
     );
-
-    // TODO: Implement this test!
+    const form = await $('form');
```

Comment:
Nice, this test looks good. Generally we want to standardize the libraries we use in a single file (matching the first test makes sense), so using `ExpectedConditions` instead would be preferred